dev-libs/libstrophe/metadata.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<maintainer type="person">
+		<email>hobbes@ualberta.ca</email>
+		<name>Calvin beck</name>
+	</maintainer>
```

Comment:
You'll need to add someone to commit for you. It's usually the proxy-maintainer project, but if you find a dev who commits for you, he needs to be added here.

Reply:
Hmmm. I must not have understood the proxy-maintainer thing I read. I'm not sure what you mean exactly. I'll give it another read.

Reply:
Check example from many other proxy-maintained ebuilds and you'll get the idea :)